### PR TITLE
Minor change in notification of bed allocation

### PR DESCRIPTION
**Pull request by cp-Coder**
# Updates

- [x] Changes related to #3018
- [x] Minor change in notification when there is a bed allocation error

### CODE REVIEW

**Comment by aeswibon** [comment]
PR #3075 doesn't fix things

https://user-images.githubusercontent.com/77210185/178829462-ac16cbee-cd36-46ef-bd0b-a5ae3c2f9c12.mp4

File: src/Components/Facility/Consultations/Beds.tsx [hunk 1]
```diff
@@ -90,6 +90,10 @@ const Beds = (props: BedsProps) => {
         msg: "Bed allocated successfully",
       });
       window.location.reload();
+    } else if (res && res.status !== 400) {
```
Comment (developedBySJ) [inline src/Components/Facility/Consultations/Beds.tsx:93]: Why do we need this notification?

**Comment by nihal467** [comment]
@developedBySJ @cp-Coder once the PR is ready to test, add need testing

**Comment by aeswibon** [comment]
Closing this PR as the issue has been fixed.

**Pull request closed without merging**